examples/solidstart-minimal/src/server/uploadthing.ts

```diff
@@ -0,0 +1,37 @@
+import { createUploadthing } from "uploadthing/server";
+import type { FileRouter } from "uploadthing/server";
+
+const f = createUploadthing();
+
+/**
+ * This is your Uploadthing file router. For more information:
+ * @see https://docs.uploadthing.com/api-reference/server#file-routes
+ */
+
+export const uploadRouter = {
+  videoAndImage: f({
+    image: {
+      maxFileSize: "4MB",
+      maxFileCount: 4,
+    },
+    video: {
+      maxFileSize: "16MB",
+    },
+  })
+    .middleware(({ req }) => {
+      // Check some condition based on the incoming requrest
+      if (!req.headers.get("x-some-header")) {
```

Comment:
We probably should remove this because the examples will be broken by default?

Reply:
commented it